### DISCUSSION TITLE
i have no idea how to execute bridge

### DISCUSSION THREAD

**Discussion by v3rtual** [bridge-core/bridge.#249] [Q&A] [answered]
I can't find any exe file to open, i cant find anything to open so that i can use bridge. How do i even execute it?

**Comment by solvedDev** [comment, marked as answer]
Releases: https://github.com/bridge-core/bridge./releases/

![image](https://user-images.githubusercontent.com/33347616/89227191-7d9ab100-d5dd-11ea-80cc-acb4165ac29c.png)
